Scripts/ExtractDomainFromUrlFormat/ExtractDomainFromUrlFormat_test.py

```diff
@@ -15,3 +16,12 @@
 def test_extract_domain(input, domain):
     res = extract_domain(input)
     assert res == domain
+
+
+@pytest.mark.parametrize('input,url', [
+                        ('http:example.com', 'http://example.com'),
+                        ('http:\\\\example.com', 'http://example.com'),
+                        ])
```

Comment:
Add also these urls to the above test: `test_extract_domain` so we are sure we are covered full flow.